docs/1.getting-started/4.styling.md

```diff
@@ -542,3 +542,8 @@ export default defineNuxtConfig({
   },
 })
 ```
+
+::tip
+If you are using pnpm, the actual path will be different.
+Read this [blog post](https://dev.to/dreitzner/nuxt3-lcp-advanced-optimizations-with-pnpm-12h3) to get started.
```

Comment:
I think the blog is quite short and is possible to fit into this docs page as a separate section, without the need of linking to other site.

Reply:
let's update the code above instead, so it works even on pnpm 🙏

Reply:
@danielroe thank you for the feedback. 

I pushed the change with the code directly included now.

Reply:
I'm suggesting instead that we don't have a workaround for pnpm. Instead, we update the code block above it so it works equally well on pnpm and non-pnpm projects

You can access `chunk.isEntry` IIRC

Reply:
@danielroe thank you for pointing that out.

I psuhed the changes